core/domain/email_manager.py

```diff
@@ -227,9 +227,12 @@ def log_new_error(*args, **kwargs):
         '- The Oppia Contributor Dashboard Team'
         '<br><br>%s'
     ),
-    'email_subject': 'Contributor Dashboard Reviewer Opportunities',
+    'email_subject': 'Contributor Dashboard Reviewer Opportunities'
+}
+
+HTML_FOR_REVIEWABLE_SUGGESTION_EMAIL_INFO = {
```

Comment:
Why _INFO? We talked about this before.

Also, perhaps HTML_FOR_REVIEWABLE_SUGGESTION_EMAILS suffices.

Reply:
Because the object is a ReviewableSuggestionEmailInfo object. Alternatively I can do HTML_FOR_SUGGESTION_DESCRIPTION